src/models/VocabularyMiner.ts

```diff
@@ -71,6 +71,17 @@ export class VocabularyMiner {
     this.textToIgnore = textToIgnore
   }
 
+  getTextWords(): string[] {
+        // Split the text into sentences
+        const sentences = splitBySentence(this.text)
+        // Split each sentence into words
+        const wordsBySentence = sentences.map((sentence) =>
+          splitSentenceIntoWords(sentence)
+        )
+
+        return wordsBySentence[0]
+  }
```

Comment:
This works but I don't understand why lol why do we get the first `wordsBySentence`?